jax/numpy/lax_numpy.py

```diff
@@ -897,6 +897,46 @@ def histogram(a, bins=10, range=None, weights=None, density=None):
     counts = counts / bin_widths / counts.sum()
   return counts, bin_edges
 
+_range = range
```

Comment:
You can remove `_range`, and use `builtins.range` below.

Reply:
will do